client-mvp-04/src/utils/uiKit/uiKit.js

```diff
@@ -33,117 +33,107 @@ const DevUiKit = () => {
-        >
+        <a key={'create-link'} className="redirect-link accent-link" href={'/'}>
           CREATE ACCOUNT
         </a>
-        <p className={'dev-comment'}>
+        <p className="dev-comment">
           Accent Link: 13px, bold, $accent-royal-blue-color
         </p>
 
         <a
           key={'tooltip-link'}
-          className={'redirect-link tooltip-link'}
+          className="redirect-link tooltip-link"
           href={'/'}
         >
           More details here
         </a>
-        <p className={'dev-comment'}>
+        <p className="dev-comment">
           Tooltip Link: 10px, bold, $link-accent-color
         </p>
 
-        <a className={'join-link'} href={'/'}>
-          Join new project <span className={'join-plus'}>+</span>
+        <a className="join-link" href={'/'}>
+          Join new project <span className="join-plus">+</span>
         </a>
-        <p className={'dev-comment'}>Join Link: 24px, bold</p>
+        <p className="dev-comment">Join Link: 24px, bold</p>
 
-        <div className={'project-link'}>
+        <div className="project-link">
           <img
             src={projectIcon}
-            className={'project-link-icon'}
+            className="project-link-icon"
             alt="project-link"
           />
-          <span className={'project-link-name'}>311 Data Project</span>
+          <span className="project-link-name">311 Data Project</span>
         </div>
-        <p className={'dev-comment'}>Project Link: 24px, semi-bold</p>
+        <p className="dev-comment">Project Link: 24px, semi-bold</p>
 
         <ErrorMessage content={'*Please enter a valid email address'} />
-        <p className={'dev-comment'}>
+        <p className="dev-comment">
           Error Message: italic, semi-bold; components/common/errorMessage
         </p>
       </div>
 
       {/*** BUTTONS ***/}
-      <div className={'button-container'}>
-        <h1 className={'kit-sec-title'}>*** BUTTONS ***</h1>
+      <div className="button-container">
+        <h1 className="kit-sec-title">*** BUTTONS ***</h1>
         <Button content={`Sign In`} />
-        <p className={'dev-comment'}>
-          Primary default button, use without class
-        </p>
-        <Button content={`Sign In`} className={'btn-accent'} />
-        <p className={'dev-comment'}>
-          Accent button, use with class .btn-accent
-        </p>
-        <Button content={`Sign In`} className={'btn-square'} />
-        <p className={'dev-comment'}>
-          Square button, use with class .btn-square
-        </p>
-        <button className={'btn-confirm'}>
-          <img src={confirmIcon} className={'btn-icon'} alt={'confirm'} />
+        <p className="dev-comment">Primary default button, use without class</p>
+        <Button content={`Sign In`} className="btn-accent" />
+        <p className="dev-comment">Accent button, use with class .btn-accent</p>
+        <Button content={`Sign In`} className="btn-square" />
```

Comment:
Where the Button component has content={`Sign In`}, I don't see any reason why there needs to be {} and ``. I think the Sign In should be a string with "" and no need for {} or ``. Again, perhaps ask another more senior dev about this.

Reply:
I agree, you don't need the curly braces here either.

Reply:
I agree